ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1317,8 +1317,15 @@ class JITModBuilder {
             ConstantAsMetadata *CAM =
                 cast<ConstantAsMetadata>(IMD->getOperand(0));
             if (CAM->getValue()->isOneValue()) {
+              // The intrinsic was inlined so we don't need to expect an
+              // unmappable block and thus can just copy the call instruction
+              // and continue processing the current block.
+              copyInstruction(&Builder, cast<CallInst>(I), CurBBIdx,
+                              CurInstrIdx);
               continue;
             }
+            // The intrinsic wasn't inlined so we let the following code handle
+            // it which already knows how to deal with such cases.
```

Comment:
How do we know to expect an unmappable in this instance?

Reply:
The code will just fall through to `handleCallInstr` which deals with those cases.